references/Subgraph/SubgraphData.md

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 1
+title: Subgraph Data Introduction
+---
+
+# Balancer Subgraph Introduction
+
+The Balancer Subgraph indexes data on the Balancer smart contracts with a GraphQL interface. It updates data in response to function calls and contract events to maintain data on the `Vault`, `Pools`, `AssetManagers` etc, to power front-end apps and integrations.
+
+Balancer has a GraphQL API Endpoint hosted by [The Graph](https://thegraph.com/docs/about/introduction#what-the-graph-is) called a subgraph for indexing and organizing data from the Sablier smart contracts.
+
+This subgraph can be used to query Balancer data.
+
+Subgraph information is serviced by a decentralized group of server operators called Indexers.

```

Comment:
This is only correct for Subgraph Studio. Balancer still uses the Hosted Service, which runs on centralized servers managed by Edge&Node.

Reply:
I will remove this sentence.